eng/targets/Settings.props

```diff
@@ -17,7 +17,8 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <ToolsetPackagesDir>$(RepoRoot)build\ToolsetPackages\</ToolsetPackagesDir>
 
-    <RoslynPortableTargetFrameworks>net472;netcoreapp2.1</RoslynPortableTargetFrameworks>
+    <RoslynPortableTargetFrameworks>netcoreapp2.1</RoslynPortableTargetFrameworks>
+    <RoslynPortableTargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">$(RoslynPortableTargetFrameworks);net472</RoslynPortableTargetFrameworks>
```

Comment:
The only concern I have is that this will change the default tfm we build for in, as netcoreapp2.1 will be listed first. I think I'm fine with this, but @jaredpar for his opinion as well.

Reply:
Good catch. We need to keep net472 as the first TF listed because that controls items like which tests run by default.

Reply:
@crummel we'll need to change this.